src/error.rs

```diff
@@ -381,6 +381,9 @@ pub struct WriteConcernError {
     /// A description of the error that occurred.
     #[serde(rename = "errmsg")]
     pub message: String,
+
+    #[serde(rename = "errInfo")]
+    pub err_info: Option<Document>,
```

Comment:
Oh I actually do have one comment. According to the [spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#error-handling), this field is meant to be named `details`.

Reply:
fixed